src/libs/actions/Task.ts

```diff
@@ -575,6 +575,8 @@ function editTaskAssignee(report: OnyxTypes.Report, ownerAccountID: number, assi
 
     let assigneeChatReportOnyxData;
     const assigneeChatReportID = assigneeChatReport ? assigneeChatReport.reportID : '-1';
+    const parentReport = getParentReport(report);
+    const taskOwnerAccountID = getTaskOwnerAccountID(report);
```

Comment:
I find it strange that there is the parameter `ownerAccountID` and this new `taskOwnerAccountID`. Is `ownerAccountID` not the owner of the task?

Reply:
I see, maybe we should rename that parameter to "actorAccountID" or "sessionAccountID"? Can you choose one and rename please 🙏